query/sources/impl/src/test/groovy/org/codice/ddf/admin/sources/csw/persist/UpdateCswConfigurationSpec.groovy

```diff
@@ -169,6 +169,38 @@ class UpdateCswConfigurationTest extends Specification {
         report.messages().get(0).code == DefaultMessages.FAILED_PERSIST
     }
 
+    def 'Updating with flag password sends service properties without password'() {
+        setup:
+        updateCswConfiguration.setValue(createCswUpdateArgs(FLAG_PASSWORD))
+        serviceActions.read(_ as String) >> [(ID): TEST_SOURCENAME]
+        serviceReader.getServices(_, _) >> []
+        configurator.commit(_, _) >> mockReport(false)
+
+        def expectedUpdateConfig = getUpdateCswProps(FLAG_PASSWORD)
```

Comment:
✏️ This test looks wrong, as it appears it's expecting the `FLAG_PASSWORD` will be sent as opposed to no password.